libfreerdp/core/pkcs11/cert_info.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * PKCS #11 PAM Login Module
```

Comment:
yes there is libpam-pkcs11. Totally agree, I should have used as external dependency. I do the changes.